### PR TITLE
fix: align ip version preference for the wrapped  emulator

### PR DESCRIPTION
**Pull request by igorbernstein2**
When the  emulator wrapper is started with -Djava.net.preferIPv6Addresses=true on a machine that defaults to ipv4,
the golang emulator and the java client will pick different ip stacks and not be able to connect. This change will
use java to resolve the localhost's ip address and push it down to the golang emulator

### CODE REVIEW

**Review by kolea2 — DISMISSED** [review]
Can we add a test with that system property at both true and false to ensure it passes?

**Comment by igorbernstein2** [comment]
We basically have those tests already ... mac os kokoro jobs set the flag and everything else doesnt. And I dont think its worth adding more kokoro jobs for this

File: google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/Emulator.java [hunk 1]
```diff
@@ -98,10 +101,24 @@ public synchronized void start() throws IOException, TimeoutException, Interrupt
     }
     this.port = getAvailablePort();
 
+    // Try to align the localhost address across java  & golang emulator
+    // This should fix issues on systems that default  to  ipv4 but the  jvm is started with
```
Comment (mutianf) [inline google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/Emulator.java:105]: Just a nit: there are a few extra spaces in these 2 lines of comment?

**Review by mutianf — APPROVED** [review]

File: google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/Emulator.java (hunk 1 above)
Comment (igorbernstein2) [reply google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/Emulator.java]: fixed

**Pull request merged**